Ports/AvailablePorts.md

```diff
@@ -12,6 +12,7 @@ Please make sure to keep this list up to date when adding and updating ports. :^
 | [`curl`](curl/)                | curl                                          | 7.65.3            | https://curl.se/                                      |
 | [`dash`](dash/)                | DASH                                          | 0.5.10.2          | http://gondor.apana.org.au/~herbert/dash              |
 | [`diffutils`](diffutils/)      | GNU Diffutils                                 | 3.5               | https://www.gnu.org/software/diffutils/               |
+| [`dmidecode`](dmidecode/)      | dmidecode                                     | 3.3             | https://github.com/mirror/dmidecode                      |
```

Comment:
Fix table formatting alignment for consistency:

```suggestion
| [`dmidecode`](dmidecode/)      | dmidecode                                     | 3.3               | https://github.com/mirror/dmidecode                   |
```

Reply:
Thanks for noticing! just asking - is there a tool to align these columns correctly so I can do that next time?

Reply:
None that I'm aware of.